### PR TITLE
fix(panel): raised and bordered variants

### PR DESCRIPTION
**Pull request by anujsingla**
## What I did

1. Fixed raised and bordered variants CSS styles
2. Improved pf-panel docs to more closely match PFv4

## Testing Instructions

fixes - https://github.com/patternfly/patternfly-elements/issues/2402

### CODE REVIEW

File: .changeset/pf-panel-2402.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+"@patternfly/elements": patch
+---
+
+`<pf-panel>`:
+ - Fixed raised and bordered variants CSS styles
+ - Improved pf-panel docs to more closely match PFv4
```
Comment (bennypowers) [inline .changeset/pf-panel-2402.md:7]:
```suggestion
`<pf-panel>`: fixed raised and bordered variants
```

**Review by bennypowers — CHANGES_REQUESTED** [review]

**Comment by bennypowers** [comment]
Looks like this on Firefox:

image

**Comment by anujsingla** [comment]
> Looks like this on Firefox:
> 
> image

Sure, I will check why it is not working fine in the Firefox browser.

**Comment by bennypowers** [comment]
Looks like this in GNOME web 44 (webkit)

Screenshot from 2023-03-26 10-47-04

**Comment by anujsingla** [comment]
@bennypowers I have fixed the firefox browser issue. Please review this MR.

**Review by bennypowers — APPROVED** [review]
LGTM

**Pull request merged**